cli/azd/pkg/pipeline/pipeline_manager.go

```diff
@@ -714,6 +747,19 @@ func (pm *PipelineManager) initialize(ctx context.Context, override string) erro
 	pm.scmProvider = scmProvider
 	pm.ciProvider = ciProvider
 
+	infra, err := pm.importManager.ProjectInfrastructure(ctx, prjConfig)
+	if err != nil {
+		return err
+	}
+	defer func() { _ = infra.Cleanup() }()
+	pm.infra = infra
+
+	pm.configOptions = &ConfigOptions{
+		Variables:                    slices.Clone(prjConfig.Pipeline.Variables),
+		Secrets:                      slices.Clone(prjConfig.Pipeline.Secrets),
+		AdditionalVariablesAsSecrets: prjConfig.Pipeline.AdditionalVariablesAsSecrets,
```

Comment:
Please update `azure.yaml` schemas to support these new options.

Reply:
I was hoping to do this as a separate PR which gets merged on release-day.

Reply:
https://github.com/Azure/azure-dev/pull/3169